selfdrive/car/honda/carcontroller.py

```diff
@@ -140,12 +140,16 @@ def update(self, sendcan, enabled, CS, frame, final_gas, final_brake, final_stee
     tt = sec_since_boot()
     GAS_MAX = 1004
     BRAKE_MAX = 1024/4
-    STEER_MAX = 0xF00
+    if CS.crv:
+      STEER_MAX = 0xFFF/2
```

Comment:
I was changing a lot of variables at once during initial testing, but 0x7FF seemed to be too high. The system is working well at 0x320 now, but I will start to increase it and do more testing over time to determine the real max value.

The issue was that the system would just give up on steering for a while and let me drift right off the road. I printed out some debug information and while it was drifting, I was getting apply_steer values of either 0, -2047 or +2048. Once the values showed something other than those three, it would steer again.  Lowering down STEER_MAX down to 0x320 seemed to clear this up, but it is possible that other changes could account for it as well. More testing with fewer changes would help me clear this up.